packages/core/src/storages/dataset.ts

```diff
@@ -282,6 +284,21 @@ export class Dataset<Data extends Dictionary = Dictionary> {
         }
     }
 
+    /**
+     *
+     * Save the entirety of the dataset's contents into one file within a key-value store.
+     *
+     *
+     * @param key The name of the value to save the data in.
+     * @param options An optional options object where you can provide a `keyValueStoreName` and a `contentType` for the output.
+     */
+    async exportToValue(key: string, options: RecordOptions & { keyValueStoreName?: string } = {}) {
```

Comment:
```suggestion
    async exportToValue(key?: string, options: RecordOptions & { keyValueStoreName?: string } = {}) {
```

Reply:
imo it should be required and the fallback would be just for vanilla js users, but okie :D